frontend/packages/dev-console/src/utils/shared-submit-utils.ts

```diff
@@ -78,6 +78,13 @@ export const createService = (
     const port = { containerPort: _.toInteger(unknownTargetPort), protocol: 'TCP' };
     ports = [...ports, port];
   }
+  if (
+    unknownTargetPort &&
+    unknownTargetPort !== '8080' &&
```

Comment:
We shouldn't use hardcoded values since they could be changed, use `defaultUnknownPort` instead.